### DISCUSSION TITLE
ExplorerPatcher update checksum?

### DISCUSSION THREAD

**Discussion by as232** [valinet/ExplorerPatcher#1923] [Q&A]
Where can one find the SHA256 checksum for the latest ExplorerPatcher update? Surely an applicationt hat patches OS files / the registry has checksums to verify authenticity...?

**Comment by as232** [comment]
Any chance we can get this implemented??

**Comment by as232** [comment]
It's not like generating a SHA256 checksum on an installfile is a giant ask or demanding of much effort. Why is this not being done?

**Comment by pyrates999** [comment]
https://github.com/orgs/community/discussions/23512

**Comment by as232** [comment]
https://github.com/orgs/community/discussions/23512#discussioncomment-6602360
